google-cloud-spanner/src/main/java/com/google/cloud/spanner/PartitionedDmlTransaction.java

```diff
@@ -194,22 +194,29 @@ ExecuteSqlRequest newTransactionRequestFrom(final Statement statement, final Opt
       if (options.hasTag()) {
         requestOptionsBuilder.setRequestTag(options.tag());
       }
+      if (options.hasTransactionTag()) {
+        requestOptionsBuilder.setTransactionTag(options.transactionTag());
```

Comment:
The [official documentation](https://cloud.google.com/spanner/docs/introspection/troubleshooting-with-tags#mapping_between_api_methods_and_requesttransaction_tag) states that transaction tags not supported for read-only transactions and are supported for read-write, but there’s no mention of partitioned DML transactions.
can you please verify if transaction tags are indeed supported for pdml txn?